packages/core/src/tracing/sentrySpan.ts

```diff
@@ -193,6 +193,7 @@ export class SentrySpan implements Span {
    */
   public updateName(name: string): this {
     this._name = name;
+    this.setAttribute(SEMANTIC_ATTRIBUTE_SENTRY_SOURCE, 'custom');
```

Comment:
l: Wondering if we would want to mention this in the jsdoc?

Reply:
Good point, let me add a note

Reply:
On second thought, I'll add this but only in #14280 (which is the PR to fix this and a couple of depending things in Node). Since we always pass the `Span` interface, I need to upadet the JSDoc of the interface which we should only do once the behaviour is adjusted everywhere.